store/snap_remote.go

```diff
@@ -86,7 +81,8 @@ func (s *RemoteSnap) Info() *snap.Info {
 		Revision:    s.Revision(),
 		Type:        s.Type(),
 		Channel:     s.Channel(),
-		Description: s.Description(),
+		Summary:     s.Pkg.Title,       // XXX: should be summary when the store provides it
```

Comment:
that's what it was, until the store fixes itself not sure we can do better, we actually need this I discovered in "snap find" or use some subset of description there